.github/workflows/ci-release.yml

```diff
@@ -0,0 +1,77 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - 'release-*'
```

Comment:
Is there a way you can test this workflow?

Reply:
I was thinking about this as well. As it stands, I don't think there is. But I believe a good way (without triggering any actual releases) may be to trigger it manually via the `workflow_dispatch` event, but then just "skip" any of the real deploys that we don't want to happen.

Reply:
I think this is the only outstanding issue on this PR. Curious on what your thoughts are on the above or potentially taking a different route.

Reply:
It would be great if you could make it work in your fork and publish to your personal docker hub acct. May need to add some small overrides for names just for the test, and remove before merging.